packages/clay-css/src/scss/variables/_custom-forms.scss

```diff
@@ -191,14 +193,18 @@ $custom-control-indicator-indeterminate-border-color: $custom-checkbox-indicator
 $custom-checkbox-indicator-indeterminate-box-shadow: none !default;
 $custom-checkbox-indicator-indeterminate-color: $custom-control-indicator-checked-color !default;
 
-$custom-checkbox-indicator-icon-indeterminate: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='4' height='4' viewBox='0 0 4 4'><path stroke='#{$custom-checkbox-indicator-indeterminate-color}' d='M0 2h4'/></svg>") !default;
+$custom-checkbox-indicator-icon-indeterminate: clay-svg-url(
+	"<svg xmlns='http://www.w3.org/2000/svg' width='4' height='4' viewBox='0 0 4 4'><path stroke='#{$custom-checkbox-indicator-indeterminate-color}' d='M0 2h4'/></svg>"
+) !default;
 $custom-checkbox-indicator-icon-indeterminate-bg-size: null !default;
 
 // Custom Radio
 
 $custom-radio-indicator-border-radius: 50% !default;
 
-$custom-radio-indicator-icon-checked: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='12' height='12' viewBox='-4 -4 8 8'><circle r='3' fill='#{$custom-control-indicator-checked-color}'/></svg>") !default;
+$custom-radio-indicator-icon-checked: clay-svg-url(
+	"<svg xmlns='http://www.w3.org/2000/svg' width='12' height='12' viewBox='-4 -4 8 8'><circle r='3' fill='#{$custom-control-indicator-checked-color}'/></svg>"
+) !default;
```

Comment:
@pat270 just a thought in case someone is using it, maybe it will pass with `url()`, wouldn't we have to keep that yet? maybe just a fallback if passing `url` still use `escape-svg` here. What do you think?

Reply:
You're right, we should account for that. I'll update this pr today.

Reply:
@matuzalemsteles I left the `escape-svg()` functions inside the components. We can run `clay-svg-url` through `escape-svg` without any errors. I tried to remove all uses of it, but I think we need to wait until the next major version.

Reply:
@pat270 we can keep this on our radar to remove `escape-svg` and replace it in the next major version with no problems.